plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/TypeHelper.java

```diff
@@ -122,6 +122,10 @@ private static Type fromKuduClientType(org.apache.kudu.Type ktype, ColumnTypeAtt
                 return VarbinaryType.VARBINARY;
             case DECIMAL:
                 return DecimalType.createDecimalType(attributes.getPrecision(), attributes.getScale());
+            // TODO: add support for varchar and date types
+            case VARCHAR:
```

Comment:
Since this PR is already complex enough, figure adding support for the new types (varchar/date) could come at a later time.

Reply:
Can you please create an issue and link to it here in the todo? (and below)

Reply:
Issue here: https://github.com/trinodb/trino/issues/11009